release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.2.2 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.2.2/
+active: 2201.2.2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.2.2
+    - /downloads/swan-lake-release-notes/2201.2.2/
+    - /downloads/swan-lake-release-notes/2201.2.2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.2.2-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.2.2
+
+<em>Swan Lake 2201.2.2 is the second patch release of Ballerina 2201.2.0 (Swan Lake Update 2) and it includes a new set of bug fixes to the language server and developer tooling.</em> 
+
+## Update Ballerina
+
+**If you are already using Ballerina 2201.0.0 (Swan Lake)**, run either of the commands below to directly update to 2201.2.2 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
+
+`bal dist update` (or `bal dist pull 2201.2.2`)
+
+**If you are using a version below 2201.0.0 (Swan Lake)**, run the commands below to update to 2201.2.2.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+
+2. Run `bal dist update` ( or `bal dist pull 2201.2.2`) to update your Ballerina version to 2201.2.2.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.2.2`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/2201-0-0-swan-lake/#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### New features
+
+### Improvements
+
+### Bug Fixes
+
+## Standard library updates
+
+### New features
+
+### Improvements
+
+### Bug Fixes
+
+## Developer tools updates
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.2.2 of the repositories below.
+
```

Comment:
```suggestion

- [Language](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)
```

Reply:
Needs to go under the bug fixes section of the language section? L43,

Reply:
This should come under `## Language updates` similar to what we have done in [2201.1.2 release note](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.1.2)

Reply:
Thanks. my mistake.